src/h.js

```diff
@@ -28,8 +28,8 @@ export function h(nodeName, attributes) {
 		if ((child = stack.pop()) instanceof Array) {
 			for (i=child.length; i--; ) stack.push(child[i]);
 		}
-		else if (child!=null && child!==false) {
-			if (typeof child=='number' || child===true) child = String(child);
+		else if (child!=null && child!==true && child!==false) {
```

Comment:
@NekR , @developit, isn't it slower? There's a small difference between these two ways in Chrome (in favor of typeof), but Firefox, for example, is 2x faster with plain comparison (`=== true || === false`). 

https://esbench.com/bench/58de1a6299634800a0347ba9